docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+    "is not needed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'m1': array([-1.1 , -0.88, -0.66, -0.44, -0.22,  0.  ,  0.22,  0.44,  0.66,\n",
+       "         0.88,  1.1 ]),\n",
+       " 'I0': array([2., 1., 1., 2., 1., 3., 2., 1., 1., 1., 0.], dtype=float32),\n",
+       " 'diode': array([1., 2., 1., 2., 2., 4., 4., 2., 0., 1., 1.], dtype=float32)}"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_sscan_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Post data to bluesky as a run\n",
+    "\n",
+    "So far, we have just run `scan1` and shown the data collected.  We'll need to do\n",
+    "a bit more work to get this data into bluesky as a run.  While we might just use\n",
+    "`bps.read(scan1)` to get the data, the data would be named according to the\n",
+    "`scan1` structure.  We want more meaningful names so we re-assign the names of\n",
+    "the original objects (motor and scaler channel names).  To post *bluesky data*,\n",
+    "it must come from an ophyd `Device` (subclass).  We'll create a custom one just\n",
+    "for our setup of `scan1`, then `bps.mv()` each of the arrays.  Since we do not\n",
+    "have timestamps for each of the data points, we'll post the entire array as a\n",
+    "single event.  The event will have the timestamp from the `bps.mv()` plan stub.\n",
+    "\n",
+    "The last five steps are all descriptive.  The run is opened, the `primary`\n",
+    "stream is written with the `scan_data`, then all is buttoned up and bluesky\n",
+    "finished the run.  We define `name=\"scan1\"` so the names of the data in the databroker will match their usage here.  The `scan_data` object is only a tool to get the data from the `sscan` record into the databroker."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ophyd import Component, Device, Signal\n",
+    "\n",
+    "class ThisSscanData(Device):\n",
+    "    m1 = Component(Signal)\n",
+    "    I0 = Component(Signal)\n",
+    "    diode = Component(Signal)\n",
+    "\n",
+    "scan_data = ThisSscanData(\"\", name=\"scan1\")\n",
+    "\n",
+    "def post_sscan_data(md={}):\n",
+    "    yield from bps.open_run(md)\n",
+    "    yield from bps.create(name=\"primary\")\n",
+    "    data = get_sscan_data()\n",
+    "    yield from bps.mv(\n",
+    "        scan_data.m1, data[\"m1\"],\n",
+    "        scan_data.I0, data[\"I0\"],\n",
+    "        scan_data.diode, data[\"diode\"],\n",
+    "    )\n",
+    "    yield from bps.read(scan_data)\n",
+    "    yield from bps.save()\n",
+    "    yield from bps.close_run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
I am lost from "so far" to here.

Reply:
I added various changes with the latest commit.  Did not resolve any conversations.  Please see if some of your questions have been resolved.

Reply:
Might take more discussion for us.  After tomorrow's discussion?